src/CoreShop/Bundle/FrontendBundle/Resources/config/services.yml

```diff
@@ -6,63 +6,75 @@ services:
         class: '%coreshop.frontend.controller.index%'
         calls:
             - [setContainer, ['@service_container']]
+            - [setTemplateConfigurator, ['@coreshop.frontend.template_configurator']]
```

Comment:
Use the `parent` attribute to define the parent controller so we don't need to add all those calls?

Reply:
done